xml/planning-architecture-alternative-rhel_compute_model.xml

```diff
@@ -6,6 +6,154 @@
   xmlns:xi="http://www.w3.org/2001/XInclude"
   xmlns:xlink="http://www.w3.org/1999/xlink">
  <title>&rhla; Compute Nodes</title>
+ <para>
+  There are several tasks you need to perform in order to install the software
+  required for &rhel; &compnode;s. Prior to this, you need to configure OBS
+  CentOS software repository on the deployer node and on the &rhel; &compnode;s
+  using the following URL:
+  https://download.opensuse.org/repositories/systemsmanagement:/Ardana:/8:/Cent
+  OS/CentOS_7.3
+ </para>
+
+  <procedure>
+   <step>
+    <para>
+     Openstack service and its components are installed on &rhel; &compnode;s
+     via virtualenv (venv). So you need to install the following venv packages
+     from OBS CentOS repository on the deployer node:
+     <itemizedlist>
+      <listitem>
+       <para>
+	venv-openstack-nova-rhel-x86_64
+       </para>
+      </listitem>
+      <listitem>
+       <para>
+	venv-openstack-nova-rhel-x86_64
+      </para>
+      </listitem>
+      <listitem>
+       <para>
+	venv-openstack-neutron-rhel-x86_64
+      </para>
+      </listitem>
+      <listitem>
+       <para>
+	venv-openstack-monasca_agent-rhel-x86_64
+      </para>
+      </listitem>
+     </itemizedlist>
+    </para>
+    <para>
+     On the deployer node, OBS CentOS repository needs to be configured with
+     lower priority with respect to &sles; repositories. This is needed so that
+     packages present in both repos are preferably installed from SLES repo.
```

Comment:
from SLES repo -> from _the \&slsa;_ repo*sitory*
(or maybe \&cloudos; instead of \&slsa;)